app/logic/events.py

```diff
@@ -114,10 +114,14 @@ def attemptSaveEvent(eventData, attachmentFiles = None):
     try:
         events = saveEventToDb(newEventData)
         if attachmentFiles:
+            # print("################################")
+            # print([e for e in events])
+            # print("################################")
```

Comment:
Remove the comments. These aren't useful for understanding the code, so they shouldn't be in your final version.